frontend/packages/topology/src/actions/provider.ts

```diff
@@ -13,7 +14,8 @@ export const useTopologyWorloadActionProvider = (element: GraphElement) => {
   }, [element]);
 
   return useMemo(() => {
-    if (element.getType() !== TYPE_WORKLOAD) return [[], true, undefined];
+    if (element.getType() !== TYPE_WORKLOAD && element.getType() !== NodeType.KnService)
```

Comment:
Need a better solution for this. you can import the `getModifyApplicationAction` in knative package if that can resolve the bad dependency